demo/scripts/column-chart.js

```diff
@@ -105,19 +105,30 @@ var fixtures = {
         { date: new Date('9/30/05'), value: 1.03},
         { date: new Date('12/30/05'), value:     1.348}
     ],
+    day : [ //this one doesn't work - only graphs 1 day. Not sure we should be using this anyway for days...
+        { date: new Date('3/02/05'), value:      0.583},
+        { date: new Date('3/03/05'), value: 1.027},
+        { date: new Date('3/04/05'), value: 1.03},
+        { date: new Date('3/05/05'), value:     1.348}
+    ],
+    hour : [ //this one doesn't work - only graphs 1 day. Not sure we should be using this anyway for hours...
+        { date: new Date('3/02/05'), value:      0.583},
+        { date: new Date('3/03/05'), value: 1.027},
+        { date: new Date('3/04/05'), value: 1.03},
+        { date: new Date('3/05/05'), value:     1.348}
+    ],
     multiple:[
-        {
-            date: new Date('3/31/05'), value: Math.floor(Math.random() * 40) + 10, value2: 99, value3: 26
-        },
-        {
-            date: new Date('6/30/05'), value: Math.floor(Math.random() * 40) + 10, value2: 10, value3: 21
-        },
-        {
-            date: new Date('9/30/05'), value: Math.floor(Math.random() * 40) + 10, value2: 70, value3: 13
-        },
-        {
-            date: new Date('12/30/05'), value: Math.floor(Math.random() * 40) + 10, value2: 10, value3: 29
-        }
+        {date: new Date('3/31/05'), value: Math.floor(Math.random() * 40) + 10, value2: 99, value3: 26},
+        {date: new Date('6/30/05'), value: Math.floor(Math.random() * 40) + 10, value2: 10, value3: 21},
+        {date: new Date('9/30/05'), value: Math.floor(Math.random() * 40) + 10, value2: 70, value3: 13},
+        {date: new Date('12/30/05'), value: Math.floor(Math.random() * 40) + 10, value2: 10, value3: 29}
+    ],
+    stacked:[
+        {date: new Date('3/31/05'), value: 50, value2: 99, value3: 26, value4: 40, value5: 15},
+        {date: new Date('6/30/05'), value: 25, value2: 10, value3: 21, value4: 36, value5: 22},
+        {date: new Date('9/30/05'), value: 75, value2: 70, value3: 13, value4: 12, value5: 110},
+        {date: new Date('12/30/05'), value: 125, value2: 10, value3: 29, value4: 31, value5: 40},
+        {date: new Date('5/30/06'), value: 133, value2: 25, value3: 72, value4: 105, value5: 200}
```

Comment:
It's correct. I picked a date that did not fall in the Q1 of 06 and instead, fell in Q2 of 06 to make sure that it indeed name it correctly.